### DISCUSSION TITLE
REST API: How can I use this object as Table

### DISCUSSION THREAD

**Discussion by Kanary159357** [Budibase/budibase#6103] [Help] [answered]
I have REST Query that gives this format that contains object array and page property for pagination

```
{
    data: [ {object}, {object}],// object for table
    page: {
                    token,
                    limit,
               }
}
```

The problem is that I can not Bind only data with Budibase Table Component...

I know that I count transform to get only data with transformer options, but in this way it seems to not have page property...

**Comment by melohagan** [comment, marked as answer]
Hi @Kanary159357 

I'm not quite sure what the issue is that you're having, but there's an example in the docs on how to use paginated apis:

https://docs.budibase.com/docs/rest-pagination

**Reply by Kanary159357** [reply]
Umm I misunderstood Budibase's pagination.... I thought I should give pagination data from params.....

Thanks
